wgsl/index.bs

```diff
@@ -3963,11 +3967,30 @@ references to arrays. An array not behind a reference may only be indexed by a
   <tr algorithm="signed integer division">
     <td>|e1| : |T|<br>|e2| : |T|<br>|T| is [SIGNEDINTEGRAL]
     <td>|e1| `/` |e2| : |T|
-    <td>Signed integer division. [=Component-wise=] when |T| is a vector. (OpSDiv)
+    <td>Signed integer division. [=Component-wise=] when |T| is a vector.
+
+        In the scalar case, evaluates to:
+        * |e1|, when |e1| is the most negative value in |T|, and |e2| is -1.
+        * [=truncate=](|x|) otherwise, where |x| is the
+            real-valued quotient |e1|&nbsp;&div;&nbsp;|e2|.
+
+        Note:
+        The need to ensure truncation behaviour may require an implementation
+        to perform more operations than when computing an unsigned division.
+        Use unsigned division when both operands are known to have the same sign.
+
+        Produces a [=dynamic error=] when |e2| evaluates to zero.
```

Comment:
so we can say "produces a dynamic error if e1 is the most negative value" as well?

Reply:
So that's something we can debate.  If the polyfill is used then no exception is required.  Let's discuss this next week.